File: semester I/Praktikum Pemrograman I/Modul6_JovanGilbertNatamasindah_2310817310002/Python/PRAK603-2310817310002-JovanGilbertNatamasindah.py
import numpy as np

BARIS1, BARIS2 = map(int, input("Masukkan Total bilangan baris pertama dan baris kedua : ").split())

if (BARIS1 == BARIS2) :

    ELEMEN_ARRAY1 = list(map(int,input("Masukkan angka baris pertama \t: ").split()))
    ELEMEN_ARRAY2 = list(map(int,input("Masukkan angka baris kedua \t: ").split()))

    arr1 = np.array(ELEMEN_ARRAY1).reshape(1,BARIS1)
    arr2 = np.array(ELEMEN_ARRAY2).reshape(1,BARIS2)

    i = 0
    print("Hasil : ")
    while i < BARIS1 and i < BARIS2 :
        HASIL = ELEMEN_ARRAY1[i] * ELEMEN_ARRAY2[i]
        print(HASIL, end = " ")
        i += 1

else :
    print("Jumlah tidak sama")

File: semester I/Praktikum Pemrograman I/Modul6_JovanGilbertNatamasindah_2310817310002/C/PRAK603-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main() {
    int BARIS1,BARIS2,h;

    printf("Masukkan Total bilangan baris pertama dan baris kedua : ");
    scanf("%d %d", &BARIS1,&BARIS2);

    int arr1[BARIS1];
    int arr2[BARIS2];

    if(BARIS1 == BARIS2) {
        
        printf("Masukkan angka baris pertama\t: ");
        for(int i = 0 ; i < BARIS1 ; i++) {
            scanf("%d", &arr1[i]);
        }

        printf("Masukkan angka baris kedua\t: ");
        for(int i = 0 ; i < BARIS2 ; i++) {
            scanf("%d", &arr2[i]);
        }

        printf("Hasil : \n");

        int arr[h];
        for(int h = 0 ; h < BARIS1 && h < BARIS2; h++) {
            arr[h] = arr1[h] * arr2[h];
            printf("%d ", arr[h]);
        }        
    }
    
    else {
        printf("Jumlah tidak sama");
    }
}



